pkg/printers/internalversion/describe.go

```diff
@@ -732,6 +732,8 @@ func describeVolumes(volumes []api.Volume, w PrefixWriter, space string) {
 			printPortworxVolumeSource(volume.VolumeSource.PortworxVolume, w)
 		case volume.VolumeSource.ScaleIO != nil:
 			printScaleIOVolumeSource(volume.VolumeSource.ScaleIO, w)
+		case volume.VolumeSource.CephFS != nil:
+			printCephFSVolumeSource(volume.VolumeSource.CephFS, w)
```

Comment:
Also need to add `printCephFSVolumeSource` in `describePersistentVolume`. And I sent a PR #46210 to add other volume sources.